osu.Game.Rulesets.Taiko/TaikoRuleset.cs

```diff
@@ -200,7 +200,7 @@ public override string GetDisplayNameForHitResult(HitResult result)
                     return "drum tick";
 
                 case HitResult.SmallBonus:
-                    return "strong bonus";
+                    return "bonus";
```

Comment:
For now, I think simply renaming it to "Bonus" should suffice, as that's exactly what they're made to be with the classic mod, and it doesn't impact user understanding as much.